feat: add parameterized queries (#45)
influxdata/influxdb-client-swift: Examples/ParameterizedQuery/README.md

# ParameterizedQuery

> This is an example how to query data into sequence of `FluxRecord`. 

Reviewer: ```suggestion
This is an example how to query with query parameters. 
```
